### DISCUSSION TITLE
Wrapping a list into '#enum'. Is it supposed to work?

### DISCUSSION THREAD

**Discussion by johnmapeson** [typst/typst#4880] [Q&A] [answered]
This works fine:

```
+ list item 1
+ list item 2

#set enum(numbering: "a)")

+ list item a
+ list item b

#set enum(numbering: "1.")

+ list item 1
+ list item 2
```

> <img width="118" alt="image" src="https://github.com/user-attachments/assets/6feeaf74-222f-4c8d-ab89-699ade8d3783">

And here is my attempt to get the same output by wrapping the second list into `#enum`. What I'm doing wrong?

```
+ list item 1
+ list item 2

#enum(numbering: "a)")[
+ list item a
+ list item b
]

+ list item 1
+ list item 2
```

> <img width="138" alt="image" src="https://github.com/user-attachments/assets/01fff647-11a8-4fa0-bad5-20cd67f0c68d">

**Comment by laurmaedje** [comment, marked as answer]
The `enum` function takes multiple arguments, one per item. So this
```typ
#enum(numbering: "a)")[
+ list item a
+ list item b
]
```
is creating two nested lists.

What you want is to scope the set rule instead like this:
```typ
+ list item 1
+ list item 2

#[
  #set enum(numbering: "a)")
  + list item a
  + list item b
]

+ list item 1
+ list item 2
```
The indent is not necessary, but typically used.
